packages/framework/src/Model/Watchdog/Watchdog.php

```diff
@@ -4,16 +4,16 @@
 
 namespace Shopsys\FrameworkBundle\Model\Watchdog;
 
-use DateTime;
 use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Clock\Clock;
 
 /**
  * @ORM\Table(name="watchdogs")
  * @ORM\Entity
  */
 class Watchdog
 {
-    protected const VALIDITY_PERIOD = '2 years';
+    public const VALIDITY_PERIOD = '2 years';
 
```

Comment:
I would simply call `updateValidity` in the constructor and remove validUntil from the data object as it cannot be modified independently of the updatedAt field. 
(meaning I would keep protected here)